tasks_week3_omer_gertler1/include/Owl.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include "Animal.h"
+
+class Owl : public Animal
+{
+private:
+	int dir = 0;
+	enum Dir { downLeft, downRight, upLeft, upRight};
```

Comment:
the enum is common between classes, declaring it in each class is correct, put it in separate file and include it